packages/garbo/src/diet.ts

```diff
@@ -112,12 +129,22 @@ function eatSafe(qty: number, item: Item) {
     eat($item`fudge spork`);
   }
   useIfUnused($item`milk of magnesium`, "_milkOfMagnesiumUsed", 5 * MPA);
-
-  if (!eat(qty, item)) throw "Failed to eat safely";
+  withProperties(
+    {
+      autoSatisfyWithCloset: usingMoonZoneRestaurant
+        ? false
```

Comment:
Instead of using a ternary here, just do `!usingMoonZoneRestaurant && get("autoSatisfyWithCloset")`